indico/htdocs/js/indico/modules/abstracts.js

```diff
@@ -39,7 +39,33 @@
     };
 
     global.setupAbstractPage = function setupAbstractPage() {
-        $('body').on('indico:confirmed', '.judge-button', function(evt) {
+        $('body').on('indico:confirmed', '.js-comment', function(evt) {
+            evt.preventDefault();
+
+            var $this = $(this);
+            var $box = $this.parents('.i-timeline-item');
+            var $form = $box.find('form');
+            var $page = $('.management-page');
+
+            $.ajax({
+                url: $this.data('href'),
+                method: $this.data('method'),
+                complete: IndicoUI.Dialogs.Util.progress(),
+                error: handleAjaxError,
+                data: getFormParams($form),
+                success: function(data) {
+                    if (data.page_html) {
+                        var $newPage = $(data.page_html);
+                        $page.replaceWith($newPage);
+                        $page = $newPage;
+                    } else {
+                        $form.replaceWith(data.form_html);
+                    }
+                    initForms($page);
+                    showFormErrors($page);
+                }
+            });
```

Comment:
We really need to find some nicer way to do this, but I guess we can discuss it after. But if it's identical to `.js-review` then it may be worth either adding it to the selector (or use the same function).